docs/getting-started/01-overview/01.setup.md

```diff
@@ -0,0 +1,131 @@
+
+#### Create a package.json
+Start by creating a package.json file and copy the following content:
+
+```
+ {
+  "name": "CHANGE_FOR_YOUR_PROJECT_NAME",
+  "version": "1.0.0",
+  "description": "",
+  "author": "",
+  "license": "ISC",
+  "main": "src/main.ts",
+  "scripts": {
+    "prebuild": "rm -rf dist",
+    "build": "tsc -p tsconfig.json",
+    "start": "PRISTINE_LOGGING_CONSOLE_LOGGER_OUTPUT_MODE=SIMPLE ts-node ./src/express.ts"
+  },
+  "dependencies": {
+    "@pristine-ts/common": "^0.0.140",
+    "@pristine-ts/configuration": "^0.0.140",
+    "@pristine-ts/core": "^0.0.140",
+    "@pristine-ts/event": "^0.0.140",
+    "@pristine-ts/express": "^0.0.140",
+    "@pristine-ts/logging": "^0.0.140",
+    "@pristine-ts/networking": "^0.0.140",
+    "@pristine-ts/telemetry": "^0.0.140",
+    "reflect-metadata": "^0.1.13",
+    "tsyringe": "^4.6.0",
+    "uuid": "^8.3.2"
+  },
+  "devDependencies": {
+    "@types/node": "^14.14.41",
+    "ts-node": "^9.1.1",
+    "typescript": "^4.2.4"
+  }
+}
+```
+
+#### Run the following command:
+
+```
+npm install
+```
+
+#### Create a tsconfig.json
+Create a `tsconfig.json` file and paste the following content
+
+```
+ {
+  "compilerOptions": {
+    "moduleResolution": "node",
+    "target": "es6",
+    "module":"CommonJS",
+    "lib": ["es2020"],
+    "strictNullChecks": true,
+    "sourceMap": true,
+    "declaration": false,
+    "allowSyntheticDefaultImports": true,
+    "experimentalDecorators": true,
+    "emitDecoratorMetadata": true,
+    "esModuleInterop": true,
+    "removeComments": true,
+    "outDir": "dist/cjs",
+    "typeRoots": [
+      "node_modules/@types"
+    ],
+    "skipLibCheck": true
+  },
+  "include": [
+    "src"
+  ],
+  "exclude": ["node_modules", "dist"]
+}
+
+```
+
+#### Create your application module (entry point of your application)
+Create a file named `src/app.module.ts` and put the following content:
+
+```
+import {CoreModule} from "@pristine-ts/core";
+
+export const AppModule: AppModuleInterface = {
+    importServices: [],
+    importModules: [
+     CoreModule,
+    ],
+    keyname: "my_namespace.app",
+}
+```
+
+#### Create your main file
+Create a file named `src/main.ts` and put the following content:
+
+```
+import {AppModule} from './app.module';
+import {Kernel} from "@pristine-ts/core";
+
+const bootstrap = async () => {
+    const kernel = new Kernel();
+    await kernel.init(AppModule);
+}
+
+bootstrap();
+```
+
+#### Wrapping up
+That's it really, you now know how to initialize the Pristine Kernel. You could retrieve your services and have all the dependencies injected.
+
+Now, you probably have a very important question:
+
+> Ok, but how can I host it and server Http calls?
+
+You will learn this in the next section: [Hosting](02.hosting.md)
+
+
+### Real examples
+Here are some Open Source projects that are using Pristine:
+* [Pristine Business Intelligence Tool](https://github.com/magieno/pristine-business-intelligence-tool)
```

Comment:
We could also link pristine-starter that I'll finish soon

Reply:
Yes totally, I'll do it.